api/v1alpha1/flinkcluster_types.go

```diff
@@ -23,17 +23,19 @@ import (
 
 // ClusterState defines states for a cluster.
 var ClusterState = struct {
-	Creating    string
-	Running     string
-	Reconciling string
-	Stopping    string
-	Stopped     string
+	Creating         string
+	Running          string
+	Reconciling      string
+	Stopping         string
+	PartiallyStopped string
```

Comment:
What does this mean given the cluster context?

Reply:
It corresponds to cleanup action `DeleteTaskManager`.